recipe/meta.yaml

```diff
@@ -32,14 +32,14 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
   run:
     - zlib 1.2*
+    - libgcc  # [not win]
 
 test:
   requires:
-    - python 2.7.*  # [win and py27]
-    - python 3.4.*  # [win and py34]
-    - python 3.5.*  # [win and py35]
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
 
   commands:
     # Verify UNIX CLI tools.
```

Comment:
@ocefpaf - once you update this PR, you could add ``h5fc`` as a command to test here too